pyartcd/pyartcd/pipelines/prepare_release.py

```diff
@@ -0,0 +1,450 @@
+            )
+        match = re.search(
+            r"https:\/\/errata\.devel\.redhat\.com\/advisory\/([0-9]+)", result.stdout
+        )
+        advisory_num = int(match[1])
+        return advisory_num
+
+    def save_advisories(self, advisories: Dict[str, int]):
+        if not advisories:
+            return
+        repo = self.working_dir / "ocp-build-data-push"
+        shutil.rmtree(repo, ignore_errors=True)
+        # shallow clone ocp-build-data
+        cmd = [
+            "git",
+            "-C",
+            self.working_dir,
+            "clone",
+            "-b",
+            self.group_name,
+            "--depth=1",
+            self.config["build_config"]["ocp_build_data_repo_push_url"],
+            "ocp-build-data-push",
+        ]
+        _LOGGER.debug("Running command: %s", cmd)
+        subprocess.run(cmd, check=True, universal_newlines=True, cwd=self.working_dir)
+        # update advisory numbers
+        with open(repo / "group.yml", "r") as f:
+            group_config = f.read()
+        for kind, advisory in advisories.items():
+            new_group_config = re.sub(
+                fr"^(\s+{kind}:)\s*[0-9]+$", fr"\1 {advisory}", group_config, count=1, flags=re.MULTILINE
+            )
+            group_config = new_group_config
+        # freeze automation
+        group_config = re.sub(r"^freeze_automation:.*", "freeze_automation: scheduled", group_config, count=1, flags=re.MULTILINE)
+        # update group.yml
+        with open(repo / "group.yml", "w") as f:
+            f.write(group_config)
+        cmd = ["git", "-C", repo, "add", "."]
+        subprocess.run(cmd, check=True, universal_newlines=True, cwd=self.working_dir)
+        cmd = ["git", "-C", repo, "commit", "-m",
+               "Update advisories and freeze automation on group.yml"]
+        subprocess.run(cmd, check=True, universal_newlines=True, cwd=self.working_dir)
+        if not self.dry_run:
+            _LOGGER.info("Pushing changes to upstream...")
+            cmd = ["git", "-C", repo, "push", "origin", self.group_name]
+            subprocess.run(cmd, check=True, universal_newlines=True, cwd=self.working_dir)
+        else:
+            _LOGGER.warn("Would have run %s", cmd)
+            _LOGGER.warn("Would have pushed changes to upstream")
+
+    def create_and_attach_placeholder_bug(self, kind: str, advisory: int):
+        cmd = [
+            "elliott",
+            f"--working-dir={self.elliott_working_dir}",
+            f"--group={self.group_name}",
+            "create-placeholder",
+            f"--kind={kind}",
+            f"--attach={advisory}",
+        ]
+        _LOGGER.debug("Running command: %s", cmd)
+        if self.dry_run:
+            _LOGGER.warn("Would have run: %s", cmd)
+        else:
+            subprocess.run(cmd, check=True, universal_newlines=True, cwd=self.working_dir)
+
+    def sweep_bugs(
+        self,
+        statuses: List[str] = ["ON_QA", "VERIFIED"],
```

Comment:
updated today, can we do:
```suggestion
        statuses: List[str] = ["MODIFIED", "ON_QA", "VERIFIED"],
```

Reply:
suggestion applied